### PR TITLE
stats: disable EXPECT_EQ in memory integration test

### PR DESCRIPTION
**Pull request by mattklein123**
Until a real solution for https://github.com/envoyproxy/envoy/issues/12209
is found.

Risk Level: None
Testing: N/A
Docs Changes: N/A
Release Notes: N/A

### CODE REVIEW

**Comment by mattklein123** [comment]
cc @jmarantz @antoniovicente

File: test/integration/stats_integration_test.cc [hunk 1]
```diff
@@ -383,7 +383,7 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithRealSymbolTable) {
   EXPECT_MEMORY_LE(m_per_cluster, 38000); // Round up to allow platform variations.
 }
 
-TEST_P(ClusterMemoryTestRunner, MemoryLargeHostSizeWithStats) {
+TEST_P(ClusterMemoryTestRunner, DISABLED_MemoryLargeHostSizeWithStats) {
```
Comment (antoniovicente) [inline test/integration/stats_integration_test.cc:386]: Could we instead disable the EXPECT_MEMORY_EQ's but keep the EXPECT_MEMORY_LE's?

Comment (mattklein123) [reply test/integration/stats_integration_test.cc:386]: Sure

**Pull request merged**